release/models/policy-forwarding/openconfig-pf-forwarding-policies.yang

```diff
@@ -264,6 +270,46 @@ submodule openconfig-pf-forwarding-policies {
         according to the IP destination).";
     }
 
+    leaf decap-network-instance {
+      type leafref {
+        // this must be an absolute reference to allow another NI
+        // to be referenced.
+        path "/network-instances/network-instance/config/name";
+      }
+      description
+        "This is normally used with NIs (network instances) that are
+        populated by SDN controllers (e.g. via gRIBI), where we want:
+          - The decapsulation decision to be controlled by the
+            controllers.
+          - The new header after decapsulation to be looked up in
+            another NI, and the NI is chosen by fields (e.g. source IP,
+            DSCP, etc) that are not used for route resolution.
+        When this leaf is set, packets matching the match criteria for
+        the forwarding rule should be looked up in the specified NI.
+        It is expected that the NI should only contain routes that have
+        next hop action as decapsualtion.
+          - If there is a match, the packet should be decapsulated and
```

Comment:
What if a match occurs for a route with a NH action other than decapsulation (i.e., a successful LPM result is obtained but the resulting NH does not have a decap action)?  I'm assuming based on https://github.com/openconfig/public/pull/727/files#diff-6e72bad429863fd2ad7d49eb5ef687e6e158ea87b354c7582125144242b4d798R310 that we should fallback